parser/src/main/java/dev/snowdrop/parser/ParseOpenRewriteCSV.java

```diff
@@ -0,0 +1,81 @@
+package dev.snowdrop.parser;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvParser;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+
+import java.io.*;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ParseOpenRewriteCSV {
+    public static void main(String[] args) throws Exception {
+
+        String DATA_TABLES = "/Users/cmoullia/code/application-modernisation/migration-tool-parent/applications/demo-spring-boot-todo-app/target/rewrite/datatables";
```

Comment:
Will this work everywhere?

Reply:
No. I will remove this class as a new one will be used within the migration-tool and not here

Reply:
Removed: a8dddb5